projects/frontend/shared-components/cicd/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+# Copyright 2023-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+echo "Logging npm engines version..."
+npm version
```

Comment:
Is there a shared-components npm version prerequisite?

Reply:
```
  "engines": {
    "node": "^14.15.0 || >=16.10.0",
    "npm": ">=8.5.5"
  },
```

Reply:
We should be good based on the output
```
{
  'shared-wrapper': '0.0.0',
  npm: '8.10.0',
  node: '16.15.0',
  v8: '9.4.146.24-node.20',
  uv: '1.43.0',
  zlib: '1.2.11',
  brotli: '1.0.9',
  ares: '1.18.1',
  modules: '93',
  nghttp2: '1.47.0',
  napi: '8',
  llhttp: '6.0.4',
  openssl: '1.1.1n+quic',
  cldr: '[40](https://gitlab.com/vmware-analytics/versatile-data-kit/-/jobs/3856991824#L40).0',
  icu: '70.1',
  tz: '2021a3',
  unicode: '14.0',
  ngtcp2: '0.1.0-DEV',
  nghttp3: '0.1.0-DEV'
}
```

Reply:
I just added the engine-strict flag to .npmrc, seemed a lot cleaner.